src/dibbs_text_to_code/services/embed.py

```diff
@@ -0,0 +1,11 @@
+from sentence_transformers import SentenceTransformer
+from torch import Tensor
+
+from dibbs_text_to_code.configs import MODEL_NAME
+
+model = SentenceTransformer(MODEL_NAME)
+
+
+def embed(input_text: str) -> Tensor:
+    """Embed text."""
+    return model.encode(input_text)
```

Comment:
This is awesome and simple!  Should we update the doc string to be a bit more descriptive?  This may only be relevant for maybe our actual APIs though....but just checking.